emails-and-notifications/sendgrid-email/addtolist_es6.js

```diff
@@ -4,17 +4,23 @@ module.exports = (event) => {
 
   return new Promise((resolve, reject) => {
     let request = sg.emptyRequest()
+    
+    // add SG custom fields here
     const { email, nameFirst: first_name, nameLast: last_name } = event.data.User.node
     request.body = [{ email, first_name, last_name }]
     request.method = "POST"
     request.path = "/v3/contactdb/recipients"
 
-    callAPI(request).then(response => {
+    // this call creates a new recipient in Sendgrid
+    // and adds it to the global list "All contacts"
+    callAPI(sg, request).then(response => {
+      
+      // this call adds newly created recipient to a specific list (not required)
       let request = sg.emptyRequest()
       request.method = 'POST'
       request.path = '/v3/contactdb/lists/___SENDGRID_LIST_ID___/recipients/' + response.body.persisted_recipients[0]
 
-      callAPI(request)
+      callAPI(sg, request)
```

Comment:
Why not just move the `sg` declaration out of the function, instead of passing it around?

Reply:
We could indeed, tbh I just wanted to push a quick fix for the repo and myself